src/tutorial/introduction.rst

```diff
@@ -3,13 +3,6 @@
 Introduction
 ============
 
-.. TODO
-
-.. warning::
-
-   This section has not yet been upgraded to match Cylc 8 changes.
-
-
```

Comment:
Should this be deleted yet? There's still a screenshot of the Cylc <del>7</del> 6 (!) GUI below

Reply:
I think that _this_ section should be pretty OK. @oliver-sanders  has implemented functionality which will produce something similar (enough, IMO) to what is shown.

Reply:
```suggestion
.. TODO

.. warning::

   This section has not yet been upgraded to match Cylc 8 changes.

```